src/gui/widgets/multiline_text.hpp

```diff
@@ -135,6 +135,17 @@ class multiline_text : public text_box_base
 	{
 		set_cursor(get_length(), select);
 		scroll_vert_ = scrollbar_base::END;
+
+		fire(event::NOTIFY_MODIFIED, *this, nullptr);
+	}
+
+	/** Inherited from text_box_base. */
+	void goto_start_of_data(const bool select = false) override
+	{
+		set_cursor(0, select);
```

Comment:
Would it be better to call the super method instead of `set_cursor` here? Right now it makes no difference, but if there's any chance that someday you might want to change the _base_ behaviour… I'm not if this is something worth caring about.

Reply:
Yes, I've already changed them to the superclass method. Should be in the next commit.